src/viam/robot/client.py

```diff
@@ -787,3 +789,30 @@ async def get_cloud_metadata(self) -> GetCloudMetadataResponse:
 
         request = GetCloudMetadataRequest()
         return await self._client.GetCloudMetadata(request)
+
+    ############
+    # Shutdown #
+    ############
+
+    async def shutdown(self):
+        """
+        Shutdown shuts down the robot. May return DeadlineExceeded error if shutdown request times out,
+        or if robot server shuts down before having a chance to send a response. May return Unavailable error
+        if server is unavailable, or if robot server is in the process of shutting down when response is ready.
```

Comment:
(minor) This isn't formatted consistently with the SDK. Probably better to do something like
```
"""
Shuts down the robot.

Raises:
  <ErrorTypeFoo>: If {foo}
  <ErrorTypeBar>: If {bar}
"""
```

You should be able to see examples of this elsewhere in this file (see, e.g., the `get_component` method).